h2/settings.py

```diff
@@ -254,12 +254,18 @@ def __len__(self):
         return len(self._settings)
 
 
-    def __eq__(self,other):
-        return self._settings==other._settings
-
-    def __ne__(self,other):
-        return not self._settings==other._settings
-
+    def __eq__(self, other):
+        if isinstance(other, Settings):
+            return self._settings == other._settings
+        else:
+            return NotImplemented
```

Comment:
@Lukasa Huh! TIL That `NotImplemented` exists. I never knew of this construct. I will keep this in mind! Thanks! :)